libs/market-info/src/components/market-info/info-market.tsx

```diff
@@ -319,10 +318,11 @@ export const Info = ({ market, onSelect }: InfoProps) => {
           }
           assetSymbol={assetSymbol}
         >
-          <Link passHref={true} href={`/liquidity/${market.id}`}>
-            <UiToolkitLink onClick={() => onSelect(market.id)}>
-              {t('View liquidity provision table')}
-            </UiToolkitLink>
+          <Link
+            to={`/liquidity/${market.id}`}
+            onClick={() => onSelect(market.id)}
+          >
+            <UILink>{t('View liquidity provision table')}</UILink>
```

Comment:
same here?

Reply:
No react router handles the hash character as we are using the HashRouter provider